pkg/kubelet/container/runtime.go

```diff
@@ -479,6 +483,42 @@ type RuntimeStatus struct {
 	Conditions []RuntimeCondition
 }
 
+// RuntimeConfigInfo contains runtime's configuration details, eg: user-namespaces mapping between host and container
+type RuntimeConfigInfo struct {
+	UserNamespaceConfig UserNamespaceConfigInfo
```

Comment:
if this config is provided, and an ID that does not fall inside one of the mappings is provided, what is the behavior?

Reply:
this will be runtime defined.